_topic_maps/_topic_map_osd.yml

```diff
@@ -32,16 +32,42 @@ Topics:
   File: legal-notice
   Distros: openshift-dedicated
 ---
-Name: Architecture
+Name: Introduction to OpenShift Dedicated
 Dir: osd_architecture
 Distros: openshift-dedicated
 Topics:
-- Name: Introduction to OpenShift Dedicated
+- Name: Understanding OpenShift Dedicated
   File: osd-understanding
 - Name: Architecture concepts
   File: osd-architecture
+- Name: Understanding your cloud deployment options
```

Comment:
This item could potentially go in the "Getting started" book and after the restructure we could build on it to make it similar to https://docs.openshift.com/rosa/rosa_getting_started/rosa-sts-getting-started-workflow.html in the ROSA library.